lib/doekbase/data_api/tests/examples/assembly_cross_annotation.py

```diff
@@ -0,0 +1,37 @@
+
+__author__ = 'Marcin Joachimiak <mjoachimiak@lbl.gov>'
+__date__ = '11/06/15'
+
+import doekbase.data_api
+from doekbase.data_api.annotation.genome_annotation import GenomeAnnotationAPI
+import os
+
+
+def contig_gc(genome_annotation, genome="kb|g.166819"):
+​
+    assembly = genome_annotation.get_assembly()
+    print "genome_annotation.get_assembly"
+
+    
+    #get assembly info
+    assembly_details = dict()
+    assembly_details["number_of_contigs"] = assembly.get_number_contigs()
```

Comment:
That is a fair comment. However, we already have very minimal examples from
Keith's narrative. Instead of replicating, these additional examples
provide a skeleton to quickly define functions and grab different data.
assembly_details is just the data container, with easy key access. This
could be reworked, but is the general idea.

On Mon, Jan 11, 2016 at 3:37 AM, Matt Henderson <notifications@github.com>
wrote:

> In lib/doekbase/data_api/tests/examples/assembly_cross_annotation.py
> <https://github.com/kbase/data_api/pull/149#discussion_r49312079>:
>
> > +__date__ = '11/06/15'
> > +
> > +import doekbase.data_api
> > +from doekbase.data_api.annotation.genome_annotation import GenomeAnnotationAPI
> > +import os
> > +
> > +
> > +def contig_gc(genome_annotation, genome="kb|g.166819"):
> > +​
> > +    assembly = genome_annotation.get_assembly()
> > +    print "genome_annotation.get_assembly"
> > +
> > +
> > +    #get assembly info
> > +    assembly_details = dict()
> > +    assembly_details["number_of_contigs"] = assembly.get_number_contigs()
>
> assembly_details is not serving any purpose in contig_gc(), and once you
> take that out, you don't really need a contig_gc() function, because you
> are only calling get_assembly() and then get_number_contigs().
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kbase/data_api/pull/149/files#r49312079>.
>